pf4j/pom.xml

```diff
@@ -33,6 +33,12 @@
             <version>1.7.5</version>
         </dependency>
         <dependency>
+            <groupId>org.slf4j</groupId>
```

Comment:
This added dependency is an slf4j logger implementation with a **test** scope so it will allow tests to output logs. The runtime jars will NOT contain this class - as you say, the user app decides what to use.